resources/views/inc/home_messages.blade.php

```diff
@@ -0,0 +1,26 @@
+<div class="container-fluid bg-secondary">
+    @if(count($errors) > 0)
+        <br>
+        <div class="alert alert-danger">
+            <ul>
+            @foreach($errors->all() as $error)
+                <li>{{ $error }}</li>
```

Comment:
Is there a reason this needed to be copied over to the home page rather than just using what is already there?

Reply:
Yes - this ensures that the background color matches the dark homepage theme when an alert box is used versus the white theme in the dashboards.